src/status_im/switcher/switcher_container.cljs

```diff
@@ -28,7 +28,5 @@
                     :num-columns               2
                     :key-fn                    str}]]))
 
-(defn container [view-id switcher-screen-radius toggle-switcher-screen]
-  [rn/view {:style (styles/switcher-screen-container
-                    view-id switcher-screen-radius)}
-   [switch-screen toggle-switcher-screen]])
+(defn tabs [toggle-switcher-screen]
```

Comment:
Hi, currently we have only a switcher screen in the tab, that's why
  ```
  (defn tabs [toggle-switcher-screen]
    [switch-screen toggle-switcher-screen])
  ```
  But it is more like a placeholder for now, once the swipable tabs and other screens are implemented it will be something like
  ```
  (defn tabs [toggle-switcher-screen]
    [swipeable-tabs
      [switch-screen toggle-switcher-screen]
      [share-screen]
      [scan-center-screen]
      [activity-center-screen]])
  ```